system/CodeIgniter.php

```diff
@@ -979,8 +979,7 @@ protected function display404errors(PageNotFoundException $e)
         // Display 404 Errors
         $this->response->setStatusCode($e->getCode());
 
-        echo $this->outputBufferingEnd();
-        flush();
+        $this->outputBufferingEnd();
```

Comment:
Reading the code now, I don't understand why it was flushing here.
https://github.com/codeigniter4/CodeIgniter4/commit/96a0d4f3#diff-d913ae68414ad947e24e588e6ee68d68194142a8aae52e61e0aab2cc7f120e0cL982
No response headers have been sent yet.

Reply:
It looks like @iRedds was just replacing the functionality that was already there.